wolox-training/rof-angular
# Implemented history for books

## Summary

Implemented history for books

## Test Plan

* Log into your account
* Go to dashboard
* Click on any book title
* You should be redirected to the detail view of the book, showing a table with rent data.

## Trello Card 

 https://trello.com/c/r0eU6o8m

## Screenshot

![tablerent](https://cloud.githubusercontent.com/assets/2911971/10832666/729f3466-7e6b-11e5-8921-d7f552fa34a4.png)
